decred/tests/unit/dcr/test_txscript.py

```diff
@@ -2352,30 +2353,66 @@ def test_scriptNumBytes(self):
             )
 
 
-class TestVSP(unittest.TestCase):
-    def test_blob(self):
-        pi = vsp.PurchaseInfo(
-            addr="someaddr",
-            fees=1.3,
-            script=ByteArray("0a1b2c"),
-            ticketAddr="someotheraddr",
-            vBits=2,
-            vBitsVer=1,
-            stamp=123456,
-        )
-        stakePool = vsp.VotingServiceProvider(
-            "http://localhost:54321", "somerandomkey", mainnet.Name, pi
-        )
-        b = stakePool.serialize()
-        rePool = vsp.VotingServiceProvider.unblob(b.b)
-        self.assertEqual(stakePool.url, rePool.url)
-        self.assertEqual(stakePool.apiKey, rePool.apiKey)
-        self.assertEqual(stakePool.net.Name, rePool.net.Name)
-        rePI = rePool.purchaseInfo
-        self.assertEqual(pi.poolAddress, rePI.poolAddress)
-        self.assertEqual(pi.poolFees, rePI.poolFees)
-        self.assertEqual(pi.script, rePI.script)
-        self.assertEqual(pi.ticketAddress, rePI.ticketAddress)
-        self.assertEqual(pi.voteBits, rePI.voteBits)
-        self.assertEqual(pi.voteBitsVersion, rePI.voteBitsVersion)
-        self.assertEqual(pi.unixTimestamp, rePI.unixTimestamp)
+def test_get_P2PKH_pCode():
+    """
+    Args:
+        name (str): Short description of the test.
+        pkScript (ByteArray): The script.
+        wantException (Exception): If present this exception should be thrown.
+        want (int): The expected stake opcode or txscript.opNonstake.
+    """
```

Comment:
These are not invocation arguments, are they? And docstrings are not really useful in tests anyway. I suggest to make them a comment above the `tests` list.

Reply:
Docstrings on test functions are super OK, but this info does belong near the `tests` array.